tests/route/test_route_perf.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+import json
+from datetime import datetime
+from tests.common.utilities import wait_until
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
+]
+
+ROUTE_JSON = 'route.json'
+ROUTE_TIMEOUT = 20 # Time limit for applying route changes
+ROUTE_TABLE_NAME = 'ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY'
+
+def prepare_dut(duthost):
+    # Set up interface
+    duthost.shell('sudo config interface ip add Ethernet72 10.1.0.54/31')
+    
+    # Set up neighbor
+    duthost.shell('sudo ip neigh add 10.1.0.37 lladdr 55:54:00:ad:48:98 dev Ethernet72')
```

Comment:
>10.1.0.37 [](start = 37, length = 9)

The neigh IP is too similar to the interface IPs. Do you want to use a totally different IP?

Reply:
Updated to a different IP.

Reply:
@qiluo-msft , @shi-su , the neighbor is expected to be in the same subnet as interface IP. With the subnet of /31, it can have only one neighbor, ie. 10.1.0.55. Suggest to change to a /24 subnet and have few neighbors so as to cover ECMP as well.

Reply:
@prsunny . Thanks for this suggestion. I updated the test to fix it.